packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4514,6 +4517,11 @@ export type Profile = {
     displayName?: string;
 };
 
+// @public
```

Comment:
Why do we use a second profile here?

Reply:
There is profile exported from Calling, cannot have two profiles exported in callWithChat, will rename this to chatProfile